### PR TITLE
Update theme for quantity field type

### PR DESCRIPTION
**Pull request by ryanmitchell**

### CODE REVIEW

File: _partials/cartBox/item_option_quantity.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php foreach ($optionValues as $optionValue) { ?>
+    <?php
+    $optionIndex = $optionValue->menu_option_value_id;
```
Comment (sampoyigi) [inline _partials/cartBox/item_option_quantity.php:3]: Can you use an input group here with - + and align to the left to match other options?

Comment (ryanmitchell) [reply _partials/cartBox/item_option_quantity.php:3]: Yes, I can do, but I don't think it works as well layout wise. It takes up a lot of space and looks odd when you have other option types in the same popup.

See screens attached to show what I mean.

Let me know what you think is best!
Screenshot 2020-06-21 at 18 55 32
Screenshot 2020-06-21 at 18 55 00

Comment (sampoyigi) [reply _partials/cartBox/item_option_quantity.php:3]: How about reducing the width of the input without the - +?

Comment (ryanmitchell) [reply _partials/cartBox/item_option_quantity.php]: Yeah thats done and pushed.
Can you rebuild app.css - it errors for me as I dont seem to have some dependencies.

Comment (sampoyigi) [reply _partials/cartBox/item_option_quantity.php]: Thank you and will do.

**Comment by sampoyigi** [comment]
Merged!

**Pull request closed without merging**